cloudify/plugins/lifecycle.py

```diff
@@ -45,15 +51,16 @@ def reinstall_node_instances(graph,
                              ignore_failure,
                              related_nodes=None):
     uninstall_node_instances(graph, node_instances, ignore_failure,
-                             related_nodes)
+                             related_nodes, name_prefix='reinstall')
```

Comment:
maybe `reinstall-` with the dash so that the name is then `reinstall-install`, not `reinstallinsall`

Reply:
I thought about it and maybe single `re` would do?

BTW there are entries with `name` like `unlinkuninstall` or `establishinstall` in `tasks_graphs`.  Maybe we should/could do something about it as well.

Reply:
go ahead :D 
https://github.com/cloudify-cosmo/cloudify-common/blob/1d9e1e0301e3bf5378cdbc1b002a67ad9771e319/cloudify/plugins/lifecycle.py#L67
and here
https://github.com/cloudify-cosmo/cloudify-common/blob/1d9e1e0301e3bf5378cdbc1b002a67ad9771e319/cloudify/plugins/lifecycle.py#L79